lib/IRGen/GenClangType.cpp

```diff
@@ -303,12 +303,12 @@ ClangTypeConverter::reverseBuiltinTypeMapping(IRGenModule &IGM,
     // Handle Int and UInt specially. On Apple platforms, these correspond to
     // the NSInteger and NSUInteger typedefs, so map them back to those typedefs
     // if they're available, to ensure we get consistent ObjC @encode strings.
-    if (swiftType->getAnyNominal() == IGM.Context.getIntDecl()) {
+    if (swiftType->isInt()) {
```

Comment:
For changes in this file, please also update the corresponding code in `ClangTypeConverter.cpp`, so that they don't get out of sync. 🙂 See my comment [here](https://github.com/apple/swift/pull/30630#discussion_r405057467) in case you're wondering why there is this duplication.